pipeline-modules/app-service/azure/pipeline-config.yaml

```diff
@@ -59,5 +57,7 @@ template: |
       printf "{{env_key}}=%s\n" "$(printf "%s" "${{env_key}}" | jq -aRs .)" >> {{pipeline_env_file}}
       {%- endfor %}
 
+  {% if variable_group_name %}
```

Comment:
do we need this condition?
We always create variable group in azure, and even if its empty it will be equivalent to the written condition

Reply:
I added this because it was helping me running via pipeline playground.